libutempter/Makefile

```diff
@@ -21,6 +21,8 @@
 PROJECT = utempter
 VERSION = $(shell sed '/^Version: */!d;s///;q' libutempter.spec)
 MAJOR = 0
+RMFLAGS = -fr
+CC = gcc
```

Comment:
Why fix the compiler to GCC? Clang is also supported and for cross-compiling `CC` must be override-able.

Reply:
Thanks for your advice, now I stop overriding CC flag